src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PrintPreviewControl.PrintPreviewControlAccessibleObjectTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using static Interop.UiaCore;
+
+namespace System.Windows.Forms.Tests
+{
+    public class PrintPreviewControl_PrintPreviewControlAccessibleObjectTests : IClassFixture<ThreadExceptionFixture>
+    {
+        [WinFormsFact]
+        public void PrintPreviewControlAccessibleObject_Ctor_Default()
+        {
+            using PrintPreviewControl printPreviewControl = new();
+            PrintPreviewControl.PrintPreviewControlAccessibleObject accessibleObject = new(printPreviewControl);
+
+            Assert.Equal(printPreviewControl, accessibleObject.Owner);
+            Assert.False(printPreviewControl.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData((int)UIA.NamePropertyId, "TestName")]
+        [InlineData((int)UIA.AutomationIdPropertyId, "PrintPreviewControl1")]
+        public void PrintPreviewControlAccessibleObject_GetPropertyValue_Invoke_ReturnsExpected(int propertyID, object expected)
+        {
+            using var control = new PrintPreviewControl
+            {
+                Name = "PrintPreviewControl1",
+                AccessibleName = "TestName"
+            };
+
+            Assert.False(control.IsHandleCreated);
+            var accessibleObject = new PrintPreviewControl.PrintPreviewControlAccessibleObject(control);
+            object value = accessibleObject.GetPropertyValue((UIA)propertyID);
+
+            Assert.Equal(expected, value);
+            Assert.False(control.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void PrintPreviewControlAccessibleObject_GetPropertyValue_Focused_ReturnsExpected()
+        {
+            using Form form = new();
+            using var control = new PrintPreviewControl();
```

Comment:
Please use one style for your tests:
```suggestion
            using Form form = new();
            using PrintPreviewControl control = new();
```